src/frontend/src/lib/styles/global/background.scss

```diff
@@ -0,0 +1,3 @@
+.bg-primary {
```

Comment:
Do we need a class? Can't it be resolved with tailwind? Or this will be reused?

Reply:
two things for this:

1. it will be re-used
2. tailwind does not understand `bg-primary` automatically and I would prefer to leave it like this if we are going to change the primary color at some point

Reply:
1. Will we re-used the background or will we reused  or the button style?

2. `bg-[var(--color-primary))]`?

Reply:
1. the background with primary color
2. yes, initially i did that, but I thought that maybe we would prefer have a specific class; but if we don't, I for sure can do that

Reply:
I prefer not adding class. There are few that are still here for historical reason but, given that we are using tailwind, I kind of feel that we should not interfer with global style unless it's configuration. You know what I mean?

Reply:
Ah, then I will use your suggestion: done!